src/server/routes/rest.js

```diff
@@ -36,6 +37,12 @@ router.get('/get-graph-and-layout', function (req, res) {
   });
 });
 
+router.get('/test/', (req, res) => {
+  const genes = req.query.genes;
+
+  res.json(geneQueryService(genes));
```

Comment:
you should return an empty response since geneQueryService has not been implemented.  This will throw an error if you go to localhost:3000/api/test

Reply:
What's an empty response?

Reply:
return an empty object as the response.


res.json({})